### PR TITLE
monitoring: adjust thresholds for noisy critical alerts

### PR DESCRIPTION
**Pull request by bobheadxi**
Adjust thresholds for frequently firing `critical` alerts, as seen on sourcegraph.com:

* gitserver: disk_space_remaining
* precise-code-intel-bundle-manager: disk_space_remaining

Dogfooding alerting (https://github.com/sourcegraph/sourcegraph/issues/5370) will probably involve subscribing our opsgenie to all critical alerts, so I'm thinking now might be a good time to assess whether or not these are actually critical. From [our docs](https://docs.sourcegraph.com/admin/observability/alerting_custom_consumption#the-difference-between-critical-and-warning-alerts):

> Critical alerts are guaranteed to be a real issue with Sourcegraph.

I'm assuming this means that in the long term, customers (and us) should be feel safe setting up notifications for Sourcegraph without being worried about being paged for things that can safely be ignored.

Slack thread: https://sourcegraph.slack.com/archives/CJX299FGE/p1594113545255400

### CODE REVIEW

File: monitoring/git_server.go [hunk 1]
```diff
@@ -15,8 +15,8 @@ func GitServer() *Container {
 							Description:     "disk space remaining by instance",
 							Query:           `(src_gitserver_disk_space_available / src_gitserver_disk_space_total) * 100`,
 							DataMayNotExist: true,
-							Warning:         Alert{LessOrEqual: 25},
-							Critical:        Alert{LessOrEqual: 15},
+							Warning:         Alert{LessOrEqual: 15},
```
Comment (keegancsmith) [inline monitoring/git_server.go:18]: gitserver will try to maintain 10% free https://sourcegraph.com/github.com/sourcegraph/sourcegraph@a6783a73a70372dcaf6b8ee900f29b23f533834c/-/blob/cmd/gitserver/main.go#L29 So this warning will flake. I would suggest removing the warning alert and just rely on the critical alert. ie if the gitserver free space maintenance isn't working there is likely a critical condition.

Comment (bobheadxi) [reply monitoring/git_server.go:18]: does this apply for `precise-code-intel-bundle-manager` as well?

**Comment by slimsag** [comment]
> I'm assuming this means that in the long term, customers (and us) should be feel safe setting up notifications for Sourcegraph without being worried about being paged for things that can safely be ignored.

This is correct.

File: monitoring/git_server.go (hunk 1 above)
Comment (slimsag) [reply monitoring/git_server.go:18]: precise-code-intel-bundle-manager has no such similar logic, no.

Comment (bobheadxi) [reply monitoring/git_server.go]: 06d81ed988

**Comment by bobheadxi** [comment]
Merging this for now and opening https://github.com/sourcegraph/sourcegraph/issues/12011 to track other alerts we might want to adjust!

File: monitoring/git_server.go (hunk 1 above)
Comment (pecigonzalo) [reply monitoring/git_server.go]: Alternatively we can make the warning only fire if stays there for X amount of time that is higher than what we expect.

**Pull request merged**